payloads/library/general/Uptime/payload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Title:		Uptime
+# Description:		The uptime payload displays the current time, the length of time the system has been up, the number of users, and the load average of the system over the last 1, 5, and 15 minutes.
+# Author:		Moliata
+# Version:		1.0
+# Category:		general
+# Target:		Packet Squirrel by Hak5
+# Net Mode:		TRANSPARENT
+function start()
+{
+        LED SETUP
+        NETMODE TRANSPARENT
+        mkdir -p /mnt/loot/uptime
+        sleep 5
```

Comment:
why sleep here?

Reply:
Sleep is used a) not to "mess up" LED, as it would have to change in like 5 milliseconds from SETUP to ATTACK b) same thing Darren did in one of his Packet Squirrel payloads c) probably to be enough time from changing between different netmodes.

Reply:
I see, makes sense.